examples/pytorch/graphsage/experimental/README.md

```diff
@@ -45,11 +61,11 @@ Pytorch distributed requires one of the trainer process to be the master. Here w
 
 ```bash
 # run client on machine 0
-python3 -m torch.distributed.launch --nproc_per_node=1 --nnodes=4 --node_rank=0 --master_addr="172.31.16.250" --master_port=1234 train_dist.py --graph-name ogb-product --ip_config ip_config.txt --num-epochs 3 --num-client 4 --batch-size 1000 --lr 0.1
+python3 -m torch.distributed.launch --nproc_per_node=1 --nnodes=4 --node_rank=0 --master_addr="172.31.16.250" --master_port=1234 train_dist.py --graph-name ogb-product --ip_config ip_config.txt --num-epochs 3 --batch-size 1000 --lr 0.1
```

Comment:
`DGL_ROLE=client`?

Reply:
I think we should remove `--graph-name` here.

Reply:
the client needs to know the graph name. otherwise, it cannot create a DistGraph client.